website/docs/user-group-role/user/user_basic_operations.md

```diff
@@ -23,7 +20,7 @@ The following topics are for the basic management of users: how to create, modif
 
 -   **Name**: The display name of the user.
 -   **Email**: The email address of the user. That will be used if there is a [notification rule](../../events/notifications) triggered or for [email stages](../../flow/stages/email).
```

Comment:
Suggestion: "Email addresses are used in emails stages and to receive notifications, if configured."  _That_ is a hard word to use in English, and we have so many concepts floating around pronouns are treacherous.

Reply:
Nice rewrite, thank you.